test/context-propagation-only.test.js

```diff
@@ -191,7 +191,7 @@ tape.test('contextPropagationOnly', function (suite) {
       // stacktrace collection when contextPropagationOnly=true. It isn't a
       // perfect way to test that.
       const durationMs = duration[0] / 1e3 + duration[1] / 1e6
-      const THRESHOLD_MS = 3 // Is this long enough for slow CI?
+      const THRESHOLD_MS = 5 // Is this long enough for slow CI?
```

Comment:
REVIEW NOTE: This is an unrelated change. I hit a spurious test failure with this tight threshold, so I bumped the threshold to make it less likely to hit it in CI.

Reply:
For example this just failed on "main": https://github.com/elastic/apm-agent-nodejs/runs/7771135311?check_suite_focus=true

```
not ok 16 captureError is fast (<3ms): 3.015229ms
```